<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Todour Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnDirBrowse">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Inactive</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="toolTip">
        <string>Keywords that mark an item as inactive. Seperated by ';'</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Active Font</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="le_activeFont">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Active</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_fontDlgActive">
          <property name="text">
           <string>Font</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_colorDlgActive">
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Inactive Font</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLineEdit" name="le_InactiveFont">
          <property name="text">
           <string>Inactive</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_fontDlgInactive">
          <property name="text">
           <string>Font</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_colorDlgInactive">
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_separate">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Sort all inactives last</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_autorefresh">
     <property name="text">
      <string>Refresh automatically when files has changed</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_dates">
     <property name="text">
      <string>Add dates to lines</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_showdates">
     <property name="text">
      <string>Show dates</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_liveSearch">
     <property name="text">
      <string>Live search</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_hotKey">
     <property name="text">
      <string>Ctrl+Alt+t as Todo hotkey</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_deletedfile">
     <property name="toolTip">
      <string>Save deleted items in deleted.txt</string>
     </property>
     <property name="text">
      <string>Track delete</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
